server/src/internalClusterTest/java/org/elasticsearch/threadpool/SimpleThreadPoolIT.java

```diff
@@ -148,15 +149,17 @@ public void testThreadPoolMetrics() throws Exception {
             assertNoFailures(prepareSearch("idx").setQuery(QueryBuilders.termQuery("str_value", "s" + i)));
             assertNoFailures(prepareSearch("idx").setQuery(QueryBuilders.termQuery("l_value", i)));
         }
+
         final var tp = internalCluster().getInstance(ThreadPool.class, dataNodeName);
+        final var tps = new AtomicReference<ThreadPoolStats>();
```

Comment:
Does this actually need to be atomic for consistency, or is it just a holder? (in which case you could just use a `ThreadPoolStats[1]`)

Reply:
Just a holder 👍 Thanks, I'll use `ThreadPoolStats[1]` to make that clear